### ISSUE TITLE
Grid content sizing bug (no interop between Firefox and Chrome)

### ISSUE THREAD

**Issue by kumarharsh** [rachelandrew/gridbugs#22]
I just ran into a bug relating while creating a grid like this:

```
-----------------------------
|          Header           |
-----------------------------
|   Nav  |      Content     |
|        |                  |
-----------------------------
```

The grid has a set size, say `200px`.
The `Nav` has a height which depends on it's children. Should the nav exceed the grid size, scrollbars must appear within the nav.

This codepen illustrates the issue:
https://codepen.io/khs/pen/pWWWxz

* Firefox displays nav properly when a `min-height:100%` is used in CSS, whereas in Chrome/Opera the nav breaks out of the grid boundaries.
* The reverse happen when `height:100%` is set on the nav (and no min-height) - broken in Firefox, OK in Chrome/Opera.
* When `max-height:100%` is set, both Firefox and Chrome behave correctly, but Chrome seemingly doesn't update the Nav's height properly causing layout issues. If I change the window size, the sizing corrects itself. This is a different bug, which I'm still trying to reproduce reliably.

This issue is also filed here: https://bugzilla.mozilla.org/show_bug.cgi?id=1381512

**Comment by kumarharsh** [comment]
So, reading up more on how grid is implemented, it seems like assuming `fr` to mean Total height of grid *minus* height of other rows is wrong.

Both the issues in Bugzilla (https://bugzilla.mozilla.org/show_bug.cgi?id=1381512), and Chromium tracker (https://bugs.chromium.org/p/chromium/issues/detail?id=756859) are closed as wontfix. 

The explanation given by Chromium team is that 1fr actually means `minmax(auto, 1fr)` - meaning if the content is large enough, the height will be `auto`, thus any children of the grid-item with `height: 100%` will not work as expected.

Digging some more, it seems like this was discussed in one of CSS-WG meetings, with similar outcomes:
https://github.com/w3c/csswg-drafts/issues/1777

With these details known, would it be useful to document the behaviour of 1fr on this site @rachelandrew?

**Comment by rachelandrew** [comment]
Yes, I intend to once that discussion (which you'll see I'm part of) is resolved.
